views/templates/front/payment-method.tpl

```diff
@@ -27,26 +27,52 @@
     </form>
 {else}
     <div class="row adyen-payment {$paymentMethodType|escape:'html':'UTF-8'}"
-         data-local-payment-method="{$paymentMethodType|escape:'html':'UTF-8'}"
-    >
+         data-local-payment-method="{$paymentMethodType|escape:'html':'UTF-8'}">
+        {if $isPrestaShop16}
+        <div class="col-xs-12 col-md-12">
+            <div class="payment_module" style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$paymentMethodBrand|escape:'html':'UTF-8'}.png)">
+                {* If collapsing is NOT disabled due to config *}
+                <span {if empty($removeCollapse)} class="adyen-payment-method-label adyen-collapser collapsed" data-toggle="collapse" data-target="#collapse{$paymentMethodType|escape:'html':'UTF-8'}" aria-expanded="false" aria-controls="collapse{$paymentMethodType|escape:'html':'UTF-8'}"
```

Comment:
In case you know exactly what can be the 2 values of the $enableCollapse then it's easier to read if you are checking for that and not if it's empty.

```suggestion
                <span {if $enableCollapse === true} class="adyen-payment-method-label adyen-collapser collapsed" data-toggle="collapse" data-target="#collapse{$paymentMethodType|escape:'html':'UTF-8'}" aria-expanded="false" aria-controls="collapse{$paymentMethodType|escape:'html':'UTF-8'}"
```

Reply:
The empty should have been removed before the committed changes, since the checks work without it